Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -619,28 +885,58 @@ public class KotlinTokenizer: SwiftTokenizer {
         case let .interpolatedString(_, rawText):
             return tokenizeInterpolatedString(rawText, node: expression)
         case .array(let exprs):
-            return
-                expression.newToken(.identifier, "listOf") +
-                expression.newToken(.startOfScope, "(") +
-                exprs.map { tokenize($0) }.joined(token: expression.newToken(.delimiter, ", ")) +
-                expression.newToken(.endOfScope, ")")
+            // Trying to output example mutableListOf<String> if it's a declartion, or mutableListOf("xx", "yy") if it's an instance with values, not great
+            var hasType = false
+            for e in exprs {
+                switch e {
+                case let _ as IdentifierExpression:
+                    hasType = true
+                default:
+                    break
+                }
+            }
+            if hasType && exprs.count == 1 && !firstCharIsUpper(str:exprs[0].description) {
+                hasType = false
+            }
+            let middle = exprs.map { tokenize($0) }.joined(token: expression.newToken(.delimiter, ", "))
+            let mutable = [ expression.newToken(.identifier, "mutableListOf") ]
+            if hasType {
+                return
+                    mutable +
+                    [ expression.newToken(.startOfScope, "<") ] +
+                    middle +
+                    [ expression.newToken(.endOfScope, ">") ]
+            } else {
+                return
+                    mutable +
+                        [ expression.newToken(.startOfScope, "(") ] +
+                        middle +
+                        [ expression.newToken(.endOfScope, ")") ]
+            }
+            // have to output mutableMap since all swift arrays are mutable. Not great.
+//                expression.newToken(.identifier, "listOf") +
+//                expression.newToken(.startOfScope, "(") +
+//                exprs.map { tokenize($0) }.joined(token: expression.newToken(.delimiter, ", ")) +
+//                expression.newToken(.endOfScope, ")")
         case .dictionary(let entries):
             return
-                expression.newToken(.identifier, "mapOf") +
-                expression.newToken(.startOfScope, "(") +
+                expression.newToken(.identifier, "mutableMapOf") +
+                expression.newToken(.startOfScope, "<") +
                 entries.map { tokenize($0, node: expression) }
                     .joined(token: expression.newToken(.delimiter, ", ")) +
-                expression.newToken(.endOfScope, ")")
+                expression.newToken(.endOfScope, ">")
```

Comment:
I think this change not work properly right? the type info should go before the `(`and the list of entries, but now the entries will go inside the `<>`